src/features/persoon/persoon-zoeken/PersonenOverzicht.vue

```diff
@@ -50,14 +50,17 @@
 import DutchDate from "@/components/DutchDate.vue";
 import type { Persoon } from "@/services/brp";
 import { useRouter } from "vue-router";
-import { mutate } from "swrv";
-import { watchEffect } from "vue";
+import { ref, watchEffect } from "vue";
 import { useSystemen } from "@/services/environment/fetch-systemen";
 import {
   useContactmomentStore,
   type ContactmomentKlant,
 } from "@/stores/contactmoment";
-import { fetchKlantByKlantIdentificatorOk } from "@/features/klant/klant-details/fetch-klant";
+import {
```

Comment:
de ene feature maakt gebruik van functies van een andere feature. is dit tijdelijk? verwachten we dat dit uiteindelijk weg kan? kunnen we er dan een comment bij zetten?

Reply:
goed gezien. ik heb wat verschoven. 901a6e33d640dc3e72ca52b2e9726c9b2dd65e18